src/workflows-app/components/SubmissionDetailsInputsTable.ts

```diff
@@ -0,0 +1,120 @@
+import { useState } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { AutoSizer } from 'react-virtualized';
+import { FlexTable, Sortable, tableHeight, TextCell } from 'src/components/table';
+import { getInputTableData, InputTableData } from 'src/workflows-app/utils/submission-utils';
+
+import { RecordLookupInputSource } from '../models/submission-models';
+
+const rowWidth = 100;
+const rowHeight = 50;
+
+const SubmissionDetailsInputsTable = ({ configuredInputDefinition }) => {
```

Comment:
This component could use a type for its props.
```suggestion
interface SubmissionDetailsInputsTableProps {
  configuredInputDefinition: ???
}

const SubmissionDetailsInputsTable = (props: SubmissionDetailsInputsTable): ReactNode => {
```